### PR TITLE
[formrecognizer] Doc fixes

### PR DESCRIPTION
**Pull request by catalinaperalta**

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_analysis_client.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class DocumentAnalysisClient(FormRecognizerClientBase):
     methods based on inputs from a URL and inputs from a stream.
 
     .. note:: DocumentAnalysisClient should be used with API versions
-        v2021-09-30-preview and up. To use API versions <=v2.1, instantiate a FormRecognizerClient.
+        2021-09-30-preview and up. To use API versions <=v2.1, instantiate a FormRecognizerClient.
```
Comment (kristapratico) [inline sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_analysis_client.py:28]: I think we should make this change on the rest of the clients if we make it here.

Comment (catalinaperalta) [reply sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_analysis_client.py:28]: Done

Comment (catalinaperalta) [reply sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_analysis_client.py:28]: Done

**Review by kristapratico — APPROVED** [review]

File: sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_analysis_client.py [hunk 2]
```diff
@@ -40,6 +40,9 @@ class DocumentAnalysisClient(FormRecognizerClientBase):
         <=v2.1, instantiate a FormRecognizerClient.
     :paramtype api_version: str or ~azure.ai.formrecognizer.DocumentAnalysisApiVersion
 
+    .. versionadded:: 2021-09-30-preview
+        The *DocumentAnalysisClient* and its client methods.
```
Comment (kristapratico) [inline sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_analysis_client.py:44]: add to async client?

**Pull request merged**